project/hooks/post_gen_project.py

```diff
@@ -115,6 +125,13 @@ def main():
                 context.get("devops_gha_deploy")
             ),  # {{ cookiecutter.devops_gha_deploy }}
         ],
+        [
+            handle_remove_devops_gitlab,
+            "Remove GHA deployment files",
```

Comment:
The naming is confusing. The handle says remove gitlab, but the message says remove github actions (GHA).